### PR TITLE
Test branch: (don't merge) new inline link styling

### PR DESCRIPTION
**Pull request by pjlee11**
Testing setup for https://github.com/bbc/psammead/pull/707

This is a branch for easily testing the new link styling in storybook by running the below command from psammead root directory : 
`
npm run ci:packages && cd packages/components/psammead-caption && npm link ../psammead-inline-link && cd ../psammead-paragraph && npm link ../psammead-inline-link && cd ../../../ && npm run build && npm run storybook
`

Then as per testing notes in 707 check the stories `Caption -> containing inline link` and `Paragraph -> containing inline link`. 

NB: It's not worth checking the inline-link story itself because in has no typography styling so will look buggy.

Also of note: We currently only have the Reith font-family stack setup so many of the world service language/scripts look bugged even though they have the correct typography styling. I've opened an [issue](https://github.com/bbc/psammead/issues/735) to address the font-family stacks for WS. 
The services I've highlighted which have obvious bugs are `bengali burmese guarati marathi sinhala telugu vietnamese` 

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

**Comment by jamesbrumpton** [comment]
LGTM

File: packages/components/psammead-caption/src/index.stories.jsx [hunk 1]
```diff
@@ -41,8 +41,9 @@ storiesOf('Components|Caption', module)
       ],
       ([linkText, captionText], script) => (
         <Caption script={script}>
-          {captionText}
-          <InlineLink href="https://www.bbc.com"> {linkText}</InlineLink>.
+          {`${captionText} `}
+          <InlineLink href="https://www.bbc.com">{linkText}</InlineLink>.
```
Comment (pjlee11) [inline packages/components/psammead-caption/src/index.stories.jsx]:
```suggestion
          <InlineLink href="https://www.bbc.com">{linkText}</InlineLink>
```

remove hardcoded fullstop

File: packages/components/psammead-paragraph/src/index.stories.jsx [hunk 2]
```diff
@@ -13,4 +14,18 @@ storiesOf('Components|Paragraph', module)
...
     { notes, knobs: { escapeHTML: false } },
+  )
+  .add(
+    'containing an inline link',
+    inputProvider(
+      [{ name: 'Paragraph' }, { name: 'Inline link' }],
+      ([paragraph, linkText], script) => (
+        <Paragraph script={script}>
+          {`${paragraph} `}
+          <InlineLink href="https://www.bbc.com">{linkText}</InlineLink>.
```
Comment (pjlee11) [inline packages/components/psammead-paragraph/src/index.stories.jsx]:
```suggestion
          <InlineLink href="https://www.bbc.com">{linkText}</InlineLink>
```

remove hardcoded fullstop

**Comment by pjlee11** [comment]
Following the testing confirmation on this, we can now review and merge https://github.com/bbc/psammead/pull/707. After that we can review, test and merge https://github.com/bbc/psammead/pull/745 and https://github.com/bbc/psammead/pull/746

**Pull request closed without merging**